### ISSUE TITLE
`atan` versus `atan2`

### ISSUE THREAD

**Issue by SheetJSDev** [pshihn/rough#166]
Both dashed and zigzag use `Math.atan`:

```ts
      const alpha = Math.atan((p2[1] - p1[1]) / (p2[0] - p1[0]));
```

It would seem that a generated vertical line would cause errors.  Is there a reason for preferring this over the safer `atan2` alternative?

```ts
      const alpha = Math.atan2((p2[1] - p1[1]), (p2[0] - p1[0]));
```

**Comment by pshihn** [comment]
You're correct atan2 would be the better option here, because I don't check for the denominator to be nonzero.

**Comment by pshihn** [comment]
I think I have been getting away with it because it is unlikely in zigzag lines, and the randomness in the dashed lines.
